appeals/api/src/database/seed/data-static.js

```diff
@@ -29,27 +29,27 @@ import {
  *
  */
 export const appealTypes = [
```

Comment:
With long text removed, the formatting can be the same for all.

Reply:
I don't understand. Which formatting?

Reply:
I'm referring to inside of this array, lines 34-38 and 47-51, they can be one-liners like the others.

Reply:
There's some auto formatting on commit which splits long lines...

Reply:
they're no longer long lines

Reply:
gotcha!